src/FxCop/Desktop.Analyzers/CSharp/CSharpDesktopAnalyzers.csproj

```diff
@@ -100,8 +100,9 @@
   </ItemGroup>
   <ItemGroup />
   <ItemGroup>
+    <Compile Include="Security\DoNotUseInsecureCryptographicAlgorithms.cs" />
     <Compile Include="Shared\SyntaxNodeHelper.cs" />
-    <Compile Include="Security\DoNotCatchCorruptedStateExceptionsAnalyzer.cs" />
+    <Compile Include="Security\DoNotCatchCorruptedStateExceptions.cs" />
```

Comment:
I agree that file name should be consistent with type name. That being said, it seems this is a problem for most of if not all analyzers under FxCop directory. I will address this issue with the clean-up/re-organization of the repo later.